chef/cookbooks/neutron/templates/default/dnsmasq-neutron.conf.erb

```diff
@@ -0,0 +1,4 @@
+# this file is managed by chef to give extra options to dnsmasq
+<% if @low_mtu -%>
+dhcp-option-force=26,1458
```

Comment:
I guess they found that it worked and did not want to spend time trying many different values to find the largest working one.